server/core/src/https/views/reset.rs

```diff
@@ -204,11 +205,40 @@ impl Display for PasskeyClass {
     }
 }
 
+async fn end_session_response(
+    state: ServerState,
+    kopid: KOpId,
+    client_auth_info: ClientAuthInfo,
+    jar: CookieJar,
+) -> axum::response::Result<Response> {
+    let is_logged_in = state
+        .qe_r_ref
+        .handle_auth_valid(client_auth_info, kopid.eventid)
+        .await
+        .is_ok();
+
+    if is_logged_in {
+        Ok((
+            jar,
+            HxLocation::from(Uri::from_static(Urls::Profile.as_ref())),
+            "",
+        )
+            .into_response())
+    } else {
+        Ok((
+            jar,
+            HxLocation::from(Uri::from_static(Urls::Login.as_ref())),
+            "",
+        )
+            .into_response())
```

Comment:
Makes it a little cleaner 😄 
```suggestion
    let url = if state
        .qe_r_ref
        .handle_auth_valid(client_auth_info, kopid.eventid)
        .await
        .is_ok()
    {
        Urls::Profile.as_ref()
    } else {
        Urls::Login.as_ref()
    };

    Ok((jar, HxLocation::from(Uri::from_static(url)), "").into_response())
```